### DISCUSSION TITLE
Copy source

### DISCUSSION THREAD

**Discussion by HeyItsMicro** [wstyres/Zebra#1714] [Ideas]
I think it would be useful to copy a source’s link by holding down on it in the sources tab and then having it copy to your clipboard. Or like sileo where if you hold down on a source it comes up with the copy button.

**Comment by eve0415** [comment]
From beta1.2 and up, you can go to the source tab and swipe to the right to copy the source's link. The latest beta (8a) is broken ATM, so you can downgrade to previous version or wait for an update to fix this.
Otherwise you can use Zebra1.1 and hold down on one of the source and will show the copy button.
